### ISSUE TITLE
Congested footer

### ISSUE THREAD

**Issue by Nandini80** [PranabKumarSahoo/sql-dev-tools#61]
The limited space allocated to the footer is compromising its presentability, suggesting the need for adjustments to provide it with a more visually appealing and organized appearance.

<img width="960" alt="image" src="https://github.com/PranabKumarSahoo/sql-dev-tools/assets/121488571/cc0448ad-9895-4c10-a005-6f4f7f7b10f9">

**Comment by PranabKumarSahoo** [comment]
Hello @Nandini80,

Thank you for bringing up this issue and suggesting improvements for the footer's appearance. Your attention to detail is appreciated.

Upon reviewing the current state of the footer, I also noticed some existing bugs in both the footer links and the sidebar links. It's possible that these issues are interconnected, as changes to the footer which is impacted the functionality of the sidebar.

Before proceeding with redesigning the footer, it would be beneficial to address these existing bugs. If you're interested, I can assign the task of resolving these issues to you. Once the bugs are fixed, we can then proceed with redesigning the footer to ensure both visual appeal and functionality.

Let me know if you're willing to take on this task, and I'll be happy to assign it to you.

Thank you!✌️

**Comment by Nandini80** [comment]
Yes, I am willing to solve these bugs. Please assign them to me.

**Comment by PranabKumarSahoo** [comment]
Hello @Nandini80,

Thank you for your willingness to solve the bugs in the footer. 
As I've already assigned this task to another contributor, I kindly ask for your patience until the existing bugs are fixed. 
Once the bugs are resolved, I'll be happy to assign the footer redesign task to you.

Your cooperation and dedication to improving our project are truly appreciated.

---
**Thank You! ✌️**

**Comment by Nandini80** [comment]
Okay I'll wait
